operations/app/src/environments/dev/main.tf

```diff
@@ -0,0 +1,26 @@
+variable "dev_name" {
+  type = string
+  description = "Name of developer (eg. cglodosky, rheft, jduff, etc.)"
+}
+
+terraform {
+  required_version = ">= 0.14"
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = ">= 2.26"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
```

Comment:
I see [in the Terraform docs](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs) this can be helpful if we don't have full permissions in Azure. Is this why this flag is enabled?

Reply:
Correct, I was unable to deploy anything without the flag.